nixos/modules/services/x11/xserver.nix

```diff
@@ -244,6 +244,14 @@ in
           "ati_unfree" "amdgpu" "amdgpu-pro"
           "nv" "nvidia" "nvidiaLegacy340" "nvidiaLegacy304"
         ];
+        # TODO(@oxij): think how to easily add the rest, like those nvidia things
+        relatedPackages = concatLists
+          (mapAttrsToList (n: v:
+            optional (hasPrefix "xf86video" n) {
+              name = removePrefix "xf86video" n;
+              prefix = "[pkgs.xf86video]";
```

Comment:
Should this generate `[pkgs.xf86video]-modesetting` somewhere in the manual or do I fail at reviewing this code?

Reply:
It would, but pkgs/servers/x11/xorg/default.nix doesn't have it. grepping the repo gives the following fact: "modesetting does not have a xf86videomodesetting package as it is included in xorgserver"

Reply:
I meant general format: `[pkgs.xf86video]nouveau`, say. I am trying to check if square brackets are intentional and what is their intent.